src/views/ProfileCreation/config.ts

```diff
@@ -3,6 +3,6 @@ import { parseUnits } from '@ethersproject/units'
 export const MINT_COST = parseUnits('1')
 export const REGISTER_COST = parseUnits('0.5')
 export const ALLOWANCE_MULTIPLIER = 5
-export const STARTER_NFT_BUNNY_IDS = ['sleepy', 'dollop', 'twinkle', 'churro', 'sunny']
+export const STARTER_NFT_BUNNY_IDS = ['5', '6', '7', '8', '9']
```

Comment:
Let's add after it?

```
 // ['sleepy', 'dollop', 'twinkle', 'churro', 'sunny']
 ```

Reply:
Let's do it later, need this out asap 😓

Reply:
Yes I am just a parrot https://github.com/ChefHutch/pancake-frontend/pull/32/files#r719592986